content/en/docs/upgrade/upgrade-faq.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "FAQ"
+keywords: "kubernetes, docker, kubesphere, jenkins, istio, prometheus"
+description: "Upgrade KubeSphere without Kubernetes"
+
+linkTitle: "FAQ"
+weight: 150
+---
+
+## How to upgrade Qingcloud CSI after upgrading?
+
+Curently Qingcloud CSI will not be upgraded by KubeKey. You can run the following command to upgrade CSI manually after finishing KubeSphere upgrade:
+
+```
+git clone https://github.com/yunify/qingcloud-csi.git
```

Comment:
How to verify whether the QingCloud CSI upgrade is successful?

Reply:
@Ma-Dan any comments here?

Reply:
@min-zh Is there any field indicate this upgrade is successful? I watch the controller and daemoset of the csi.

Reply:
Run follwing command to watch csi controller and daemonset is running
```
root@master1:~# kubectl get po -n kube-system | grep csi
csi-qingcloud-controller-56979d46cb-qk9ck   5/5     Running            0          24h
csi-qingcloud-node-4s8n5                    2/2     Running            0          24h
csi-qingcloud-node-65dqn                    2/2     Running            0          24h
csi-qingcloud-node-khk49                    2/2     Running            0          24h
csi-qingcloud-node-nz9q9                    2/2     Running            0          24h
csi-qingcloud-node-pxr56                    2/2     Running            0          24h
csi-qingcloud-node-whqhk                    2/2     Running            0          24h
```

Then run this command to check csi image version is 1.2.x
```
root@master1:~# kubectl get po -n kube-system csi-qingcloud-controller-56979d46cb-qk9ck -ojson | jq '.spec.containers[].image' | grep qingcloud
"csiplugin/csi-qingcloud:v1.2.0-rc.4"
```

Reply:
updated